src/Form/Form.test.js

```diff
@@ -0,0 +1,103 @@
+afterEach(() => {
+    document.body.innerHTML = null
+})
+
+describe('Test the form class reading values correctly', () => {
```

Comment:
Regression testing for form the submission handler. We want to make sure we are always reading the values correctly.